src/pkg/Microsoft.TargetingPack.Private.CoreRT/linux/Microsoft.TargetingPack.Private.CoreRT.pkgproj

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
We should have just one targeting pack based on the Windows bits. Similar to what we do for CoreCLR.